core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-input/ai-image-prompt-input.component.html

```diff
@@ -0,0 +1,96 @@
+                    fill="#8D92A5"
+                    fill-rule="evenodd" />
+                <path
+                    d="M28.6935 19.2671L22.8007 21.0668C22.6478 21.1134 22.5136 21.2101 22.4181 21.3425C22.3227 21.4749 22.2711 21.6357 22.2711 21.8011C22.2711 21.9664 22.3227 22.1273 22.4181 22.2597C22.5136 22.392 22.6478 22.4887 22.8007 22.5353L28.6935 24.3351L30.4277 30.4505C30.4726 30.6091 30.5658 30.7484 30.6933 30.8474C30.8209 30.9465 30.9759 31 31.1352 31C31.2945 31 31.4495 30.9465 31.577 30.8474C31.7045 30.7484 31.7977 30.6091 31.8427 30.4505L33.5776 24.3351L39.4704 22.5353C39.6233 22.4887 39.7575 22.392 39.8529 22.2597C39.9484 22.1273 40 21.9664 40 21.8011C40 21.6357 39.9484 21.4749 39.8529 21.3425C39.7575 21.2101 39.6233 21.1134 39.4704 21.0668L33.5776 19.2671L31.8427 13.1517C31.7978 12.993 31.7046 12.8538 31.577 12.7547C31.4495 12.6556 31.2945 12.6021 31.1352 12.6021C30.9759 12.6021 30.8208 12.6556 30.6933 12.7547C30.5658 12.8538 30.4726 12.993 30.4277 13.1517L28.6935 19.2671Z"
+                    fill="#7042F0" />
+                <path
+                    d="M35.9026 8.29011L39.0994 7.31284C39.2522 7.26623 39.3864 7.16951 39.4819 7.03716C39.5773 6.9048 39.6289 6.74394 39.6289 6.57862C39.6289 6.41331 39.5773 6.25244 39.4819 6.12008C39.3864 5.98773 39.2522 5.89101 39.0994 5.8444L35.9026 4.8679L34.9609 1.54959C34.916 1.39096 34.8228 1.25169 34.6953 1.15262C34.5677 1.05354 34.4127 1 34.2534 1C34.0941 1 33.9391 1.05354 33.8115 1.15262C33.684 1.25169 33.5908 1.39096 33.5459 1.54959L32.6049 4.8679L29.4075 5.8444C29.2546 5.891 29.1204 5.98771 29.025 6.12006C28.9295 6.25241 28.8779 6.41329 28.8779 6.57862C28.8779 6.74395 28.9295 6.90483 29.025 7.03718C29.1204 7.16953 29.2546 7.26624 29.4075 7.31284L32.6049 8.29011L33.5459 11.6076C33.5908 11.7663 33.684 11.9056 33.8115 12.0046C33.9391 12.1037 34.0941 12.1572 34.2534 12.1572C34.4127 12.1572 34.5677 12.1037 34.6953 12.0046C34.8228 11.9056 34.916 11.7663 34.9609 11.6076L35.9026 8.29011Z"
+                    fill="#7042F0" />
+                <path
+                    d="M19.6845 9.67937L16.459 11.0447C16.3233 11.1021 16.2072 11.2002 16.1254 11.3265C16.0437 11.4527 16 11.6014 16 11.7535C16 11.9056 16.0437 12.0542 16.1254 12.1805C16.2072 12.3067 16.3233 12.4048 16.459 12.4623L19.6845 13.8276L21.0001 17.175C21.0555 17.3159 21.1501 17.4364 21.2717 17.5212C21.3934 17.606 21.5366 17.6514 21.6832 17.6514C21.8297 17.6514 21.973 17.606 22.0946 17.5212C22.2163 17.4364 22.3108 17.3159 22.3662 17.175L23.6818 13.8276L26.9067 12.4623C27.0424 12.4048 27.1585 12.3067 27.2402 12.1805C27.322 12.0542 27.3656 11.9056 27.3656 11.7535C27.3656 11.6014 27.322 11.4527 27.2402 11.3265C27.1585 11.2002 27.0424 11.1021 26.9067 11.0447L23.6818 9.67937L22.3662 6.33266C22.3108 6.19183 22.2163 6.07132 22.0946 5.98648C21.973 5.90165 21.8297 5.85635 21.6832 5.85635C21.5366 5.85635 21.3934 5.90165 21.2717 5.98648C21.1501 6.07132 21.0555 6.19183 21.0001 6.33266L19.6845 9.67937Z"
+                    fill="#7042F0" />
```

Comment:
Shouldn't we use an icon from PrimeNg? Maybe we can check this with Melissa

Reply:
I asked, and she designed that icon.